apps/ai-content-generator/src/locations/Dialog.tsx

```diff
@@ -29,7 +29,7 @@ const Dialog = () => {
   };
 
   return (
-    <GeneratorProvider entryId={entryId} prompt={featureConfig[feature].prompt}>
+    <GeneratorProvider feature={feature} entryId={entryId} prompt={featureConfig[feature].prompt}>
```

Comment:
I feel since we're giving full access to the feature here, we can remove the `prompt` property